shared/src/androidTest/kotlin/org/hyperskill/step_quiz/AndroidStepQuizTest.kt

```diff
@@ -84,9 +84,9 @@ class AndroidStepQuizTest {
 
             val reducer = StepQuizReducer(
                 when (stepRouteClass) {
-                    StepRoute.Learn::class -> StepRoute.Learn(step.id)
+                    StepRoute.Learn::class -> StepRoute.Learn.Step(step.id)
                     StepRoute.LearnDaily::class -> StepRoute.LearnDaily(step.id)
-                    StepRoute.Repeat::class -> StepRoute.Repeat(step.id)
+                    StepRoute.Repeat::class -> StepRoute.Repeat.Practice(step.id)
                     StepRoute.StageImplement::class -> StepRoute.StageImplement(step.id, 1, 1)
```

Comment:
`Learn` & `Repeat` are sealed interfaces with 2 inheritors. Dedicated `when`s over `Learn` & `Repeat` `sealedSubclasses` should be added.